core/src/main/java/org/mskcc/cbio/portal/dao/DaoTypeOfCancer.java

```diff
@@ -131,20 +131,22 @@ public static int getCount() throws DaoException {
       }
    }
 
-   public static void deleteAllRecords() throws DaoException {
-      Connection con = null;
-      PreparedStatement pstmt = null;
-      ResultSet rs = null;
-      try {
-         con = JdbcUtil.getDbConnection(DaoTypeOfCancer.class);
-         pstmt = con.prepareStatement("TRUNCATE TABLE type_of_cancer");
-         pstmt.executeUpdate();
-      } catch (SQLException e) {
-         throw new DaoException(e);
-      } finally {
-         JdbcUtil.closeAll(DaoTypeOfCancer.class, con, pstmt, rs);
-      }
-   }
```

Comment:
The indentation had 3 spaces, so IntelliJ converted it to 4 spaces proper indentation.